src/Horse.CGI.pas

```diff
@@ -0,0 +1,74 @@
+unit Horse.CGI;
+
+interface
+
+uses IdHTTPWebBrokerBridge, Horse.API, IdContext;
+
```

Comment:
Added shortcut in uses of THorseRequest and THorseResponse, same as Horse.Core file.